postprocessing/timavg/timavg.csh

```diff
@@ -1,5 +1,29 @@
 #!/bin/tcsh -f
 #
+#***********************************************************************
+#                   GNU Lesser General Public License
+#
+# This file is part of the GFDL FRE NetCDF tools package (FRE-NCTools).
+#
+# FRE-NCTools is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or (at
+# your option) any later version.
+#
+# FRE-NCTools is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with FRE-NCTools (LICENSE.md).  If not, see
+# <http://www.gnu.org/licenses/>.
+#***********************************************************************
+#
+# Copyright (C) NOAA Geophysical Fluid Dynamics Laboratory, 2000-2010
+# Designed and written by V. Balaji, Amy Langenhorst and Aleksey Yakovlev
+#
```

Comment:
Do we need to extend the date from 2010 to current.

Reply:
Good question. I myself do not know at this time.

Reply:
The first year in the copyright should be the year the file was created.  Then each year the file was touched, a new year should be added.  Thus `2000-2010` indicates the files was touched in each year from 2000 to 2010.  If years are skipped, they should be comma separated: e.g. `2000-2010, 2015, 2019`.

I think it may be beneficial to start a NOAA-GFDL documents repository similar to [Google's Open Source Docs ](https://opensource.google/docs/).  The [Google Copyright Notice Usage](https://opensource.google/docs/copyright/) is relevant to this discussion.  I think, as is mentioned in the Google Copyright page, adding a copyright is optional.  However, at a minimum we should ensure all code created at GFDL (by Contractors, Feds, COOP Institutes) should have the year the file was created.

Reply:
So I wont modify timavg.csh to add to specifically a additional years. 

Adding the creation year code may be a particularly good idea for NCtools code and other code that have abandoned their original repositories.  I hope we can do that in a separate effort.

Reply:
Is it possible/doable to use the git history to retroactively add years that the files were touched? (I have been guilty in not updating the copyright text when I have made small updates, and probably others have as well.)

Reply:
Resolution moved to this issue:Adding the years has been moved to this issue:
 https://github.com/NOAA-GFDL/FRE-NCtools/issues/46

Reply:
Future work on this is moved to this issue: https://github.com/NOAA-GFDL/FRE-NCtools/issues/46